sklearn/cross_validation.py

```diff
@@ -61,6 +62,16 @@
            'train_test_split']
 
 
+class FitFailedWarning(FitFailedWarning_):
```

Comment:
No this should be `FitFailedWarning` so people can import this just like before, but with a deprecation warning of course.